.rubocop.yml

```diff
@@ -44,6 +44,9 @@ Lint/UnusedBlockArgument:
     - 'vendor/**/*'
     - '**/.irbrc'
 
+RSpec/NestedGroups:
+  Enabled: false
```

Comment:
I approve.  I think this rule is not well suited to the current state of this project.